lxd/storage/backend_lxd.go

```diff
@@ -1453,7 +1453,7 @@ func (b *lxdBackend) RefreshCustomVolume(projectName string, srcProjectName stri
 					return err
 				}
 
-				volSize, err = drivers.BlockDiskSizeBytes(volDiskPath)
+				volSize, err = shared.BlockDiskSizeBytes(volDiskPath)
```

Comment:
I'm quite keen to keep things out of `shared` unless we have a good reason to. The reason for this is that eventually we want to move to a more modern Go layout where most of the code is in the `internal` directory, so really we should be seeking to move stuff out of `shared` rather than into it.

Also `shared` the top level package is kind of a dumping ground for stuff atm, and isn't well organised.

Instead how about having a `lxd/storage/block` package with util(s) in there for block devices, similar to `lxd/storage/filesystem` we have already.

lxd-migrate can then import that (as its not released as Apache 2.0).

Reply:
Thanks. I've moved only that function into `lxd/storage/block` as suggested. I wanted to move more functions from `shared`, which seems particularly dangerous, as I am not aware what other products may reference those functions. Therefore, we can move more functionality into new package later.

Reply:
sounds good thanks